boxsdk/client/client.py

```diff
@@ -1399,7 +1398,7 @@ def get_pending_collaborations(self, limit=None, offset=None, fields=None):
         )
 
     @api_call
-    def downscope_token(self, scopes, item=None, additional_data=None):
+    def downscope_token(self, scopes, item=None, shared_link=None, additional_data=None):
```

Comment:
Also, the shared_link parameter should be after additional_data to avoid a breaking change.